homeassistant/components/vlc/media_player.py

```diff
@@ -34,6 +34,7 @@
     {
         vol.Optional(CONF_ARGUMENTS, default=""): cv.string,
         vol.Optional(CONF_NAME): cv.string,
+        vol.Optional(CONF_UNIQUE_ID): cv.string,
```

Comment:
We should not expand YAML configurations. Why can't you use the VLC Telnet integration instead?

Reply:
This integration is only for a local Python environment, so it probably wouldn't make sense to add a config flow to it. I've been using this to test out media players, and thought I would share.
It seems kind of silly to leave the integration almost non-functional without a unique id.

Reply:
This violates our ADRs, it should migrate to a config flow instead.